app/controllers/users_controller.rb

```diff
@@ -15,17 +25,56 @@ def create
     @user = User.new user_params
     if @user.save
       log_in @user
-      remember user
       flash[:success] = t(".success")
       redirect_to @user
     else
       render :new
     end
   end
 
+  def update
+    if @user.update(user_params)
+      flash[:success] = t(".success")
+      redirect_to @user
+    else
+      flash[:danger] = t(".danger")
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t(".success")
+    else
+      flash[:danger] = t(".danger")
+    end
+    redirect_to users_url
```

Comment:
Phân biệt url với path
```suggestion
    redirect_to users_path
```